condacolab.py

```diff
@@ -338,12 +339,14 @@ def check(prefix: os.PathLike = PREFIX, verbose: bool = True):
     pymaj, pymin = sys.version_info[:2]
     sitepackages = f"{prefix}/lib/python{pymaj}.{pymin}/site-packages"
     assert sitepackages in sys.path, f"💥💔💥 PYTHONPATH was not patched! Value: {sys.path}"
+    assert "/usr/local/" not in sys.path, "💥💔💥 There is some problem with installation!"
```

Comment:
```suggestion
    assert all(not path.startswith("/usr/local/") for path in sys.path), f"💥💔💥 PYTHONPATH include system locations: {[path for path in sys.path if path.startswith('/usr/local')]}!"
```

We want to ensure none of the paths in `sys.path` starts with '/usr/local'